apps/prairielearn/src/tests/fileEditor.test.js

```diff
@@ -617,9 +617,7 @@ function verifyEdit(
     assert.isString(locals.file_edit_orig_hash);
   });
   it('should have a script with draft file contents', function (callback) {
-    elemList = locals.$('script');
-    for (let i = 0; i < elemList.length; i++) {
-      let elem = elemList[i];
+    for (const elem of Array.from(locals.$('script'))) {
```

Comment:
Naturally jQuery-like things aren't actually iterables, hence the `Array.from(...)`.